.eslintrc.js

```diff
@@ -123,7 +127,7 @@ const baseConfig = {
   overrides: [
     {
       // TypeScript config
-      files: ['**/*.ts', '**/*.tsx'],
+      files: [`**/*.{${tsExtensions}}`],
```

Comment:
these arrays need joining yeah?

Reply:
Nope, JavaScript 🪄 

![toString](https://user-images.githubusercontent.com/3297808/220818290-12f3e915-db07-443d-afef-08b7ac9e217f.png)

Reply:
The template string will call the array's `toString`, which will just result in `"ts,cts,mts,tsx"`.

Reply:
Sorry, i thought these had to be pipe delimited. My bad